### PR TITLE
chore: optimize update in ubaClient

### PR DESCRIPTION
**Pull request by james-a-morris**

### CODE REVIEW

File: src/clients/UBAClient/UBAClientWithRefresh.ts [hunk 1]
```diff
@@ -25,38 +25,26 @@ export class UBAClientWithRefresh extends BaseUBAClient {
...
-      await updateUBAClient(
-        this.hubPoolClient,
-        this.spokePoolClients,
-        this.chainIdIndices,
-        this.tokens,
-        forceClientRefresh,
-        this.maxBundleStates
-      )
-    );
+  public async update(state: UBAClientState, forceClientRefresh?: boolean): Promise<void> {
```
Comment (pxrl) [inline src/clients/UBAClient/UBAClientWithRefresh.ts:28]:
```suggestion
  public async update(state?: UBAClientState, forceClientRefresh?: boolean): Promise<void> {
```

Comment (pxrl) [inline src/clients/UBAClient/UBAClientWithRefresh.ts:28]: It's confusing to only use `forceClientRefresh` if `state` is undefined.

Comment (james-a-morris) [reply src/clients/UBAClient/UBAClientWithRefresh.ts]: Done https://github.com/across-protocol/sdk-v2/pull/320/commits/ee401eb44159ae3458ae43dfa986e6a2236a8996

Comment (james-a-morris) [reply src/clients/UBAClient/UBAClientWithRefresh.ts]: Followed up offline

Comment (pxrl) [reply src/clients/UBAClient/UBAClientWithRefresh.ts]: I'm skeptical about the UX of this, but you're right that it was pre-existing, so fair enough :+1:

**Review by pxrl — APPROVED** [review]

**Pull request merged**